### PR TITLE
Pal 991 remove data gen

### PR DESCRIPTION
**Pull request by ac74475**

### CODE REVIEW

File: deployment/README.md [hunk 1]
```diff
@@ -19,7 +19,7 @@
 A collection of scripts for making life easier.
 
 ## [hr-data-generator](../hr-data-generator/README.md)
```
Comment (pd104923) [inline deployment/README.md:21]: Does this header need to be changed? There isn't a README for the hr-data-generator

File: deployment/hr-data-generator/createHRData.sh [hunk 2]
```diff
@@ -15,11 +15,11 @@
 #
```
Comment (pd104923) [inline deployment/hr-data-generator/createHRData.sh:15]: I think the directory for this file should be renamed so not to include a reference to `hr-data-generator`

**Review by pd104923 — CHANGES_REQUESTED** [review]
A couple of tweaks

File: deployment/README.md (hunk 1 above)
Comment (dev930018) [reply deployment/README.md:21]: Agreed, needs to be changed to a full github URL

File: deployment/hr-data-generator/createHRData.sh (hunk 2 above)
Comment (dev930018) [reply deployment/hr-data-generator/createHRData.sh:15]: I'm happy for this to stay as-is, but change the path that's used here to point to a parent directory `../Palisade-hr-data-generator/....` or whatever it's called, similarly to how `Palisade-services` is treated

File: deployment/README.md (hunk 1 above)
Comment (developer01189998819991197253) [reply deployment/README.md:21]: updated

File: deployment/hr-data-generator/createHRData.sh (hunk 2 above)
Comment (developer01189998819991197253) [reply deployment/hr-data-generator/createHRData.sh:15]: are you refering to the link in the file?
java -cp $FILE uk.gov.gchq.syntheticdatagenerator.CreateData $@
if so, what should it be?

Comment (dev930018) [reply deployment/hr-data-generator/createHRData.sh:15]: My mistake, just need to update the bottom comment on `Failed to find hr-data-generator jar` to `Failed to find example-runner jar`

**Review by pd104923 — APPROVED** [review]

**Pull request merged**